app/views/protocols/view_details/_research_involving.html.haml

```diff
@@ -27,7 +27,7 @@
     %h5
       %strong
         = t(:protocols)[:studies][:research_involving][:header]
-        
+
   - if research_types_info.present?
```

Comment:
I thought the same thing, but I was wondering why they had the check for the parent object there